### PR TITLE
feat: Embed Settings 

### PR DESCRIPTION
**Pull request by harshilp24**

### CODE REVIEW

File: website/docs/advanced-concepts/embed-appsmith-into-existing-application.md [hunk 1]
```diff
@@ -98,40 +98,43 @@ npx http-server
 
 Next, open [http://localhost:8080/dashboard.html](http://localhost:8080/dashboard.html) and it should open the HTML File.
 
-### Modifying Layout
+### Modifying layout
 
-If you want to get your app to use the whole page in your browser, you can still change your height and width parameters like so:
```
Comment (danciaclara) [inline website/docs/advanced-concepts/embed-appsmith-into-existing-application.md:103]: @harshilp24 did you check if this is possible in the UI generated embed URL  before deleting it?

File: website/docs/advanced-concepts/embed-appsmith-into-existing-application.md [hunk 2]
```diff
@@ -98,40 +98,43 @@ npx http-server
...
+You can modify the layout of your embedded code directly from the settings of your Appsmith app. This allows you to control the appearance of your app when it's displayed on a website or another platform. 
 
-```markup
```
Comment (danciaclara) [inline website/docs/advanced-concepts/embed-appsmith-into-existing-application.md:105]: related to the comment above

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/advanced-concepts/embed-appsmith-into-existing-application.md (hunk 1 above)
Comment (harshilp24) [reply website/docs/advanced-concepts/embed-appsmith-into-existing-application.md:103]: yes, my bad,  they still need to implement the iframe using HTML code, will update this :(

File: website/docs/advanced-concepts/embed-appsmith-into-existing-application.md [hunk 3]
```diff
@@ -98,9 +98,19 @@ npx http-server
...
+<VideoEmbed host="youtube" videoId="qACSzNMsdKA" title="Modifying layout" caption="Modifying layout"/>
+
 
-If you want to get your app to use the whole page in your browser, you can still change your height and width parameters like so:
+* To change the layout, open the `app settings`.
+* Click on `Share & Embed`.
+* Update the Embed size as desired.
+* Copy the updated embed code.
+
+By following these simple steps and the code examples and explanations provided, you would be able to customize the layout of your embedded app to fit your specific needs and requirements.
```
Comment (danciaclara) [inline website/docs/advanced-concepts/embed-appsmith-into-existing-application.md:113]:
```suggestion
If you want to get your app to use the whole page in your browser, you can change your height and width parameters like shown below:

```

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/advanced-concepts/embed-appsmith-into-existing-application.md [hunk 4]
```diff
@@ -115,23 +125,29 @@ If you want to get your app to use the whole page in your browser, you can still
 </html>
 ```
 
-#### Remove Appsmith Top Bar
+#### Remove Appsmith top bar
+
+Additionally, you can also see the Appsmith toolbar on the top, you can remove this by adding **`?embed=true`** to the share URL.
```
Comment (danciaclara) [inline website/docs/advanced-concepts/embed-appsmith-into-existing-application.md:130]:
```suggestion
You can remove the Appsmith topbar with page tabs by appending **`?embed=true`** to the share URL. Alternatively, you can get this prepared URL from the `App Settings` on the canvas as shown below.
```

File: website/docs/advanced-concepts/embed-appsmith-into-existing-application.md [hunk 5]
```diff
@@ -115,23 +125,29 @@ If you want to get your app to use the whole page in your browser, you can still
...
 
-#### Remove Appsmith Top Bar
+#### Remove Appsmith top bar
+
+Additionally, you can also see the Appsmith toolbar on the top, you can remove this by adding **`?embed=true`** to the share URL. 
+
+![Remove Appsmith top bar](/img/embed_apps.png)
 
-Additionally, you can also see the Appsmith toolbar on the top, you can remove this by adding **`?embed=true`** to the share URL. Update the code to the following:
+* To remove the top bar, open the `app settings`.
```
Comment (danciaclara) [inline website/docs/advanced-concepts/embed-appsmith-into-existing-application.md:134]:
```suggestion
* Click on `App settings` in the Canvas properties .
```

**Review by danciaclara — CHANGES_REQUESTED** [review]

**Review by danciaclara — APPROVED** [review]

**Pull request merged**